### ISSUE TITLE
CoreLocation missing from platform package on MacOS

### ISSUE THREAD

**Issue by pyryjook** [JetBrains/kotlin-native#3041]
Hey!

I'm building geo location based CLI app for MacOS and was about to use the `CoreLocation` package for the purpose. 

AppCode did not initially complain (code completion even worked as expected) anything when importing the package as follows:

```kotlin
import platform.CoreLocation.CLLocationManager
```
but when trying to build the project I see this:
```

e: /path/to/my/project/src/main/kotlin/location.kt: (2, 17): Unresolved reference: CoreLocation
e: /path/to/my/project/src/main/kotlin/location.kt: (5, 23): Unresolved reference: CLLocationManager
```
I investigated it a while and noticed that the package definition is included for the iOS: https://github.com/JetBrains/kotlin-native/blob/master/platformLibs/src/platform/ios/CoreLocation.def

But the `.def` file for `CoreLocation` is missing from the OSX folder: https://github.com/JetBrains/kotlin-native/tree/master/platformLibs/src/platform/osx

Is this on purpose or could it be added there?

Thanks already in advance!

**Comment by SvyatoslavScherbina** [comment]
Thanks for the report.
To workaround the issue please add [custom interop configuration](https://kotlinlang.org/docs/reference/building-mpp-with-gradle.html#cinterop-support) to your Gradle build using the following `.def` file:
```
language = Objective-C
package = platform.CoreLocation
modules = CoreLocation

compilerOpts = -framework CoreLocation
linkerOpts = -framework CoreLocation
```

**Comment by pyryjook** [comment]
Thank you for your fast response on this. I will use the described workaround for now, thanks for pointing it out.

**Comment by SvyatoslavScherbina** [comment]
Will likely be available by default in 1.3.50.
